schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to trigger events and/or actions.
```

Comment:
can we change "trigger events and/or actions." to "generate alerts or other events."

Reply:
done